styles.css

```diff
@@ -15,13 +15,15 @@
 
 .headline {
```

Comment:
- [ ] according to project requirement kindly use inline comments while giving code review as this will make your review specific you can check this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) on how to give code review